### PR TITLE
isDevChannel method and whitelist for access

### PR DESCRIPTION
**Pull request by dvoytenko**
/cc @erwinmombay

### CODE REVIEW

**Comment by erwinmombay** [comment]
@dvoytenko LGTM, final word from @cramforce

File: build-system/tasks/presubmit-checks.js [hunk 1]
```diff
@@ -33,6 +33,9 @@ var privateServiceFactory = 'This service should only be installed in ' +
     'the whitelisted files. Other modules should use a public function ' +
     'typically called serviceNameFor.';
 
+var shouldNeverBeUsed =
+    'Usage of this API requires is not allowed - only for internal purposes.';
```
Comment (jridgewell) [inline build-system/tasks/presubmit-checks.js]: "Usage of this API is not allowed - ..."

Comment (dvoytenko) [inline build-system/tasks/presubmit-checks.js]: Done

**Comment by cramforce** [comment]
LGTM

**Pull request merged**